.circleci/s3-website-config.json

```diff
@@ -1,18 +1,18 @@
 {
-    "Bucket": "pyodide-cdn2.iodide.io",
-    "WebsiteConfiguration": {
-        "IndexDocument": {
-            "Suffix": "index.html"
+  "Bucket": "pyodide-cdn2.iodide.io",
+  "WebsiteConfiguration": {
+    "IndexDocument": {
+      "Suffix": "index.html"
+    },
+    "RoutingRules": [
+      {
+        "Condition": {
+          "KeyPrefixEquals": "latest/"
         },
-        "RoutingRules": [
-            {
-                "Condition": {
-                    "KeyPrefixEquals": "latest/"
-                },
-                "Redirect": {
-                    "ReplaceKeyPrefixWith": "v0.14.3/"
-                }
-            }
-        ]
-    }
+        "Redirect": {
+          "ReplaceKeyPrefixWith": "v0.14.3/"
```

Comment:
Does this file do anything? It's suspicious that we haven't replaced this version for quite so long...

Reply:
I completely forgot about it.

https://cdn.jsdelivr.net/pyodide/latest/full/packages.json does redirect to https://cdn.jsdelivr.net/pyodide/v0.14.3//full/packages.json. But anyway it's not a very good idea to rely on URLs that change in time for data packages, so let's just remove this "RoutingRules" sections. Thanks for catching it!